### ISSUE TITLE
[A11y] Backend: Configurations > Settings > Features

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#3362]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Backend

### Admin user?

Yes

### Issue Description

Reference Link: https://oregondigital.org/admin/features?locale=en

Logged in gray sidebar Configuration > Settings > Features

Recieved the following errors on Accessibility Insights

<img width="964" alt="Image" src="https://github.com/user-attachments/assets/d0dd1a3c-dd15-4c9e-9b30-8269a5deb0b2" />

### Suggested Fix

Fix any of the following:

Element has insufficient color contrast of 4.47 (foreground color: #ffffff, background color: #777777, font size: 9.0pt (12px), font weight: bold). Expected contrast ratio of 4.5:1

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Revalidate the page using Accessibility Insights
